### DISCUSSION TITLE
keri-dev 06/22/23

### DISCUSSION THREAD

**Discussion by m00sey** [WebOfTrust/keri#33] [Meetings]
https://us06web.zoom.us/rec/share/PTv8MBwhvSO26YWFp92sk09SIai6B0RGeKTNzdDxLAxlfUfi4HS02uC4RN4xWPG0.fyw5Xx0iq65wxdsG?startTime=1687442610000
Passcode: 44n+qin?

keripy:

Serder class updates, table for packet types and said verification for packet type properties.
Still working on tests.
Allows us to version!

signifyts:
list credentials
challenge response and contacts
registries

keria:

signifypy:
updates for KSN changes

cesride:
attempt to re use cesride 

Potential to update with some changes from Finema:
https://github.com/WebOfTrust/kara

Dart KERI:
https://pub.dev/packages/keri supports signing at the

**Comment by SmithSamuelM** [comment]
https://github.com/WebOfTrust/kara

Kassh  also example of Sally like presentation verifier  but Sally is production GLEIF. Not accepting pull requests but modify KARA or create new repo to fork Sally if want to augment.
